geonode/base/api/views.py

```diff
@@ -196,10 +196,18 @@ def featured(self, request):
         ```
         {
             "resource_types": [
-                "layer",
-                "map",
-                "document",
-                "service"
+                {
+                    "name": "layer",
+                    "count": 5
+                },
+                {
+                    "name": "map",
+                    "count": 5
+                },
+                {
+                    "name": "document",
+                    "count": 5
```

Comment:
@marthamareal what an unusual default. Why 5? I'd put None, so it will be easier to identify problems, don't you think?

Reply:
I think using a holder eg 
"count": <number of layers>  since count will be an integer from 0 and above

Reply:
I don't like it very much since it will make the troubleshooting hard. I would use a None, so that if something goes wrong it will be easier to spot. Not a casual number for sure!